### DISCUSSION TITLE
Zeek and Sigma

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#5643] [Unsupported Versions] [answered]
Question? Is it possible to create a Sigma rule to alert on a specific zeek log?

**Comment by clairmont32** [comment, marked as answer]
Yes. This is convered in the docs (https://docs.securityonion.net/en/latest/playbook.html#playbook) and SO Essentials training video (https://youtu.be/IS2SOlDedPc). The entire training playlist they provide for free is at https://www.youtube.com/playlist?list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH and is a very good foundation to the entire set of capabilities SO provides.
